src/opera/test/util/test_metadata_utils.py

```diff
@@ -42,26 +42,36 @@ def test_get_geographic_boundaries_from_mgrs_tile_nominal(self):
         """Reproduce ADT results from values provided with code"""
         lat_min, lat_max, lon_min, lon_max = get_geographic_boundaries_from_mgrs_tile('15SXR')
 
-        self.assertAlmostEqual(lat_min, 31.616027943130398)
-        self.assertAlmostEqual(lat_max, 32.6212369766609)
-        self.assertAlmostEqual(lon_min, -91.94552881416524)
-        self.assertAlmostEqual(lon_max, -90.76425651871281)
+        self.assertAlmostEqual(lat_min, 31.572733739486036)
+        self.assertAlmostEqual(lat_max, 32.577473659397235)
+        self.assertAlmostEqual(lon_min, -91.99766472766642)
+        self.assertAlmostEqual(lon_max, -90.81751155385777)
 
     @skipIf(not osr_is_available(), reason="osgeo.osr is not installed on the local instance")
     def test_get_geographic_boundaries_from_mgrs_tile_leading_T(self):
         """Test MGRS tile code conversion when code starts with T"""
         lat_min, lat_max, lon_min, lon_max = get_geographic_boundaries_from_mgrs_tile('T15SXR')
 
-        self.assertAlmostEqual(lat_min, 31.616027943130398)
-        self.assertAlmostEqual(lat_max, 32.6212369766609)
-        self.assertAlmostEqual(lon_min, -91.94552881416524)
-        self.assertAlmostEqual(lon_max, -90.76425651871281)
+        self.assertAlmostEqual(lat_min, 31.572733739486036)
+        self.assertAlmostEqual(lat_max, 32.577473659397235)
+        self.assertAlmostEqual(lon_min, -91.99766472766642)
+        self.assertAlmostEqual(lon_max, -90.81751155385777)
```

Comment:
I see why these changed (from lines 191 and 192 in the new function).  But it is a bit concerning that our lat/lon values seem to differ from the bounding polygons from the Sentinel-2 UTM Tiling Grid (HLS is supposed to be using the exact same tiling grid).  You can find a kml of that tiling grid here:  https://eatlas.org.au/data/uuid/f7468d15-12be-4e3f-a246-b2882a324f59

Grepping through that file for the tile IDs of the ones used in the unit tests (my grep commands copied below) shows slightly different lat/lon bounds.  I assume this is due to us using a slightly different projection for our calculations.  If these outputs are only used for showing bounding boxes in Earthdata Search, say, then probably not a big deal.  But worth a conversation with ADT.
```
grep -A 20 "15SXR" S2A_OPER_GIP_TILPAR_MPC__20151209T095117_V20150622T000000_21000101T000000_B00.kml
grep -A 20 "60VXQ" S2A_OPER_GIP_TILPAR_MPC__20151209T095117_V20150622T000000_21000101T000000_B00.kml
```